astrosat_users/views/views_customers.py

```diff
@@ -64,9 +78,16 @@ class CustomerCreateView(CustomerViewMixin, generics.CreateAPIView):
     lookup_field = "id"
     lookup_url_kwarg = "customer_id"
 
-    permission_classes = [IsAuthenticated]
+    permission_classes = [IsAuthenticated, HasPendingCustomer]
     serializer_class = CustomerSerializer
 
+    def perform_create(self, serializer):
+        customer = serializer.save()
+        user = self.request.user
+        if user.pending_customer is True:
```

Comment:
This is more for my learning, In Java and JavaScript, we can do `if user.pending_customer` and drop of the `is True` bit. Is this the case in python?

Reply:
Yes, that is the case.  I prefer being explicit unless there is a reason not to.  Type `import this` in a Python shell to see why.

Reply:
Long as it's not a `fork bomb`.....